examples/satellite-azure/variables.tf

```diff
@@ -77,7 +85,7 @@ variable "instance_type" {
 variable "satellite_host_count" {
   description = "The total number of Azure host to create for control plane. "
   type        = number
-  default     = null
+  default     = 3
```

Comment:
This is defaulted to `null` now b/c we deprecated these old host_count variables in favor of the new ones which take objects.  That allows us to do multi-flavor hosts for control plane and additional hosts.

Reply:
Reverted back!